SECURITY.md

```diff
@@ -6,25 +6,41 @@ This is the security notice for all GDS (Government Digital Service) repositorie
 
 ## Reporting a Vulnerability
 
-GDS is an advocate of responsible vulnerability disclosure. If you’ve found a vulnerability, we would like to know so we can fix it.
+GDS is an advocate of responsible vulnerability disclosure. If you’ve found a vulnerability, we would like to know so we can fix it. Quick links can be found at our [security.txt](https://vdp.cabinetoffice.gov.uk/.well-known/security.txt).
 
-Send details to [disclosure@digital.cabinet-office.gov.uk], including:
+You can [report a vulnerability to GDS using HackerOne](https://hackerone.com/44c348eb-e030-4273-b445-d4a2f6f83ba8/embedded_submissions/new), please include:
 - the website, page or repository where the vulnerability can be observed
 - a brief description of the vulnerability 
+- details of the steps we need to take to reproduce the vulnerability
 - optionally the type of vulnerability and any related [OWASP category]
 - non-destructive exploitation details
+- screenshots or logs if you have them
 
-GDS use ZenDesk as a ticketing system, so you may get a reply or response from that platform.
+Alternatively, you can send details to [disclosure@digital.cabinet-office.gov.uk]. GDS use ZenDesk as a ticketing system, so you may get a reply or response from that platform.
 
 ## Scope
 The following are **not** in scope:
-- volumetric vulnerabilities, for example overwhelming a service with a high volume of requests 
-- reports indicating that our services do not fully align with “best practice”, for example missing security headers
+- break the law
+- access unnecessary or excessive amounts of data
+- modify data
+- use high-intensity invasive or destructive scanning tools to find vulnerabilities
+- try a denial of service - for example overwhelming a service on GOV.UK with a high volume of requests
+- disrupt GOV.UK’s services or systems
+- tell other people about the vulnerability you have found until we have disclosed it
+- social engineer, phish or physically attack our staff or infrastructure
+- demand money to disclose a vulnerability
 
-If you are not sure, you can still reach out via email; [disclosure@digital.cabinet-office.gov.uk].
+Only submit reports about exploitable vulnerabilities through HackerOne.
+
+If you are not sure, or you have found:
+- a non-exploitable vulnerability
+- something you think could be improved - for example, missing security headers
+- TLS configuration weaknesses - for example weak cipher suite support or the presence of TLS1.0 support
+
+Then you can still [reach out via email](disclosure@digital.cabinet-office.gov.uk) or using the [GOV.UK contact form](https://www.gov.uk/contact/govuk).
```

Comment:
It feels like this should go into the top section on Reporting a Vulnerability, rather than this section on scope?